### PR TITLE
fix: tx to byron address

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-699](https://emurgo.atlassian.net/browse/YOMO-699)

[YOMO-699]: https://emurgo.atlassian.net/browse/YOMO-699?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/package.json
Comment (stackchain) [inline apps/wallet-mobile/package.json:1]: @banklesss `cross-csl-core` was set as `devDependency` when it's actually a regular `dependency`, it also needs to be bumped to `2.6.0`

File: apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts:1]: Can we add tests to this?

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts [hunk 1]
```diff
@@ -36,16 +36,16 @@ export const getMinAmounts = async (address: Address, amounts: Balance.Amounts,
...
   )
 
-  const [value, coinsPerUtxoByte, bech32Aaddress] = await Promise.all([
+  const [value, coinsPerUtxoByte, address] = await Promise.all([
     cardanoValueFromMultiToken(multiToken),
     CardanoMobile.BigNum.fromStr(COINS_PER_UTXO_BYTE),
-    (await CardanoMobile.ByronAddress.isValid(address))
-      ? await (await CardanoMobile.ByronAddress.fromBase58(address)).toAddress()
-      : await CardanoMobile.Address.fromBech32(address),
+    (await CardanoMobile.ByronAddress.isValid(_address))
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts]: @bankless there is a `normalizeToAddress` function in the `utils` you can use it from there, it has an extra logic for `hex`, yet it also requires tests.

Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts]: i will take a look thanks

Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts]: 100% covered

Screenshot 2023-08-22 at 00 30 01

Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts]: 100% covered

Screenshot 2023-08-22 at 00 30 01

File: apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts
Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts:1]: done 100% covered

File: apps/wallet-mobile/src/yoroi-wallets/cardano/constants/common.ts [hunk 2]
```diff
@@ -57,3 +57,5 @@ export const MINIMUM_UTXO_VAL = '1000000'
 export const HISTORY_REFRESH_TIME = 25000
 
 export const COINS_PER_UTXO_WORD = '34482'
+
+export const COINS_PER_UTXO_BYTE = '4310'
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/constants/common.ts:61]: all these constants need to come from the chain, I'll ask BE team to update the tip status or create a specific endpoint for the protocol params.

**Review by stackchain — APPROVED** [review]

**Pull request merged**